manifests/v1beta1/installs/katib-openshift/kustomization.yml

```diff
@@ -0,0 +1,85 @@
+# This kustomization copies the `katib-standalone` one with following exclusions:
+#  - No Job is spawned to generate TLS key for `katib-controller` Service
+#  - Instead, the Service and WebhookConfigurations linked to it are annotated
+#    for OpenShift service controller to handle TLS certification.
+#  - PersistentVolume is not created, as OpenShift installations typically
+#    have PVC dynamical provisioning configured or need some extra administrative
+#    effort to provision a PV manually
+#  - PersistentVolumeClaim has empty `.spec.storageClassName` to allow dynamic
+#    provisioning with default StorageClass
+#
+#  Requires OpenShift version: 4.4+
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: kubeflow
+resources:
+  # Namespace.
+  - ../../components/namespace.yaml
+  # Katib controller.
+  - ../../components/controller/controller.yaml
+  - ../../components/controller/service.yaml
+  - ../../components/controller/rbac.yaml
+  - ../../components/controller/katib-config.yaml
+  - ../../components/controller/trial-templates.yaml
+  # Katib CRDs.
+  - ../../components/crd/experiment.yaml
+  - ../../components/crd/suggestion.yaml
+  - ../../components/crd/trial.yaml
+  # Katib DB manager.
+  - ../../components/db-manager/db-manager.yaml
+  - ../../components/db-manager/service.yaml
+  # Katib DB mysql.
+  - ../../components/mysql/mysql.yaml
+  - ../../components/mysql/service.yaml
+  - ../../components/mysql/pvc.yaml
+  - ../../components/mysql/secret.yaml
+  # Katib UI.
+  - ../../components/ui/ui.yaml
+  - ../../components/ui/service.yaml
+  - ../../components/ui/rbac.yaml
+  # Katib webhooks.
+  - ../../components/webhook/webhooks.yaml
```

Comment:
This section should be replaced with:
```
- ../katib-standalone/
```
This way the Kustomize `load-restrictor` flag is not necessary and the manifest can be used with Kustomize v3 and v4 more easily. The other manifests have been updated in https://github.com/kubeflow/katib/pull/1498 to also use this style of referencing rather than linking to files directly.

Reply:
@maanur @andreyvelich Given that this PR was created to fix issues with the cert-generator (https://github.com/kubeflow/katib/issues/1512), I'm not sure if it might not be better to use `katib-cert-manager` instead. You would be better judges of that.

Reply:
I think it should be up to the OpenShift users. Maybe some of them don't want to use `cert-manager` to deploy Katib.

What do you think @maanur @nakfour @gaocegege @johnugeorge  ?

Reply:
That's also what I thought. An alternative would be for Katib to require cert-manager or have that be the default, but I can understand why this wouldn't be wanted as well so that is up to you guys.

Reply:
This PR has nothing to do with cert-manager, relying on builtin OpenShift service controller instead.

Reply:
@maanur If you are not relying on the cert-generator job you should change the `resources:` section to the following:

```yaml
# Namespace.
  - ../../components/namespace
  # Katib controller.
  - ../../components/controller/
  # Katib CRDs.
  - ../../components/crd/
  # Katib DB manager.
  - ../../components/db-manager/
  # Katib DB mysql.
  - ../../components/mysql/
  # Katib UI.
  - ../../components/ui/
  # Katib webhooks.
  - ../../components/webhook/
```